deploy/kubernetes/base/gluster/slow-many.yaml

```diff
@@ -1,13 +0,0 @@
----
-apiVersion: storage.k8s.io/v1
-kind: StorageClass
-metadata:
-  name: slow-many
-parameters:
-  restauthenabled: "true"
-  resturl: http://localhost:8080
-  restuser: admin
-  secretName: heketi-admin-secret
-  secretNamespace: default
-  volumetype: replicate:2
-provisioner: kubernetes.io/glusterfs
```

Comment:
Move storage class to vortex installer.
The storage class should be applied right after kubernetes is up.